redaxo/src/addons/mediapool/pages/structure.php

```diff
@@ -63,14 +63,14 @@
                 $icat = rex_media_category::get($iid);
 
                 $n = [];
-                $n['title'] = $icat->getName();
+                $n['title'] = rex_escape($icat->getName());
                 $n['href'] = $link . $iid;
                 $breadcrumb[] = $n;
             }
             next($paths);
         }
         $n = [];
-        $n['title'] = $OOCat->getName();
+        $n['title'] = rex_escape($OOCat->getName());
```

Comment:
ich würde das escaping an der stelle machen wo auch das html erzeugt wird, nicht dort wo man die daten zusammen sammelt (da die daten ggf. auch nochmal in einem anderen context genutzt werden würden/könnten)